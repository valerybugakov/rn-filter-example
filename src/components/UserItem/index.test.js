import React from 'react'
import renderer from 'react-test-renderer'

import Component from './index'

it('renders without crashing', () => {
  const user = {
    firstName: 'Anakin',
    lastName: 'Skywalker',
    index: 1,
    key: '1',
    pressed: false,
  }

  const tree = renderer.create(<Component item={user} />).toJSON()

  expect(tree).toMatchSnapshot()
})
