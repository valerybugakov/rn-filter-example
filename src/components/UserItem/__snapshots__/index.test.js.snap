// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flexDirection": "row",
      "height": 94,
      "paddingVertical": 10,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#f5f5f5",
        "borderRadius": 22,
        "height": 44,
        "justifyContent": "center",
        "marginHorizontal": 16,
        "width": 44,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    />
  </View>
  <View
    style={
      Object {
        "alignSelf": "center",
        "flexDirection": "column",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      numberOfLines={1}
      style={
        Object {
          "fontSize": 16,
        }
      }
    >
      Anakin
       
      Skywalker
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      numberOfLines={1}
      style={
        Object {
          "color": "#a7a7a7",
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#a7a7a7",
        }
      }
    >
      last login: 
      NaN year ago
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    pointerEvents="box-only"
    rippleCentered={true}
    rippleColor="rgb(0, 0, 0)"
    rippleDuration={400}
    rippleFades={true}
    rippleOpacity={0.3}
    rippleSequential={false}
    rippleSize={0}
    style={
      Object {
        "alignItems": "center",
        "height": 32,
        "justifyContent": "center",
        "marginLeft": "auto",
        "marginRight": 20,
        "width": 32,
      }
    }
    testID={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    />
    <View
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "bottom": 0,
            "left": 0,
            "overflow": "hidden",
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "borderRadius": 24,
          },
        ]
      }
    />
  </View>
</View>
`;
